python/systematic.py

```diff
@@ -564,7 +564,7 @@ def PrepareGlobalNormalization(self,normString,abstract,chan,sam,forceNoFallback
                         abstract.hists[histName] = None
                         abstract.prepare.addHisto(histName, forceNoFallback=forceNoFallback)
 
-                    if abstract.hists[histName] is None:
+                    if abstract.readFromTree or abstract.hists[histName] is None:
```

Comment:
Just checking since I didn't follow what is going on here.  What is this fixing?

Reply:
The user systematic histograms are not initialized to None by default, so it failed the first time when running with "-t"